lib/ui/navigation/maiporarisu_navigation.dart

```diff
@@ -1,14 +1,62 @@
 import 'package:flutter/material.dart';
 
-class MaiporarisuNavigation extends StatelessWidget {
+class MaiporarisuNavigation extends StatefulWidget {
   const MaiporarisuNavigation({super.key});
 
+  @override
+  State<MaiporarisuNavigation> createState() => _MaiporarisuNavigationState();
+}
+
+class _MaiporarisuNavigationState extends State<MaiporarisuNavigation> {
+  var _currentPageIndex = 0;
+  
+  final _pages = <Widget>[
+    Container(
+      alignment: Alignment.center,
+      child: const Text('Home'),
+    ),
+    Container(
+      alignment: Alignment.center,
+      child: const Text('Schedule'),
+    ),
+    Container(
+      alignment: Alignment.center,
+      child: const Text('Setting'),
+    ),
+  ];
+
   @override
   Widget build(BuildContext context) {
-    return const Scaffold(
-      body: Center(
-        child: Text('ページ切り替え'),
+    return Scaffold(
+      bottomNavigationBar: NavigationBar(
+        onDestinationSelected: (int index) {
+          setState(() {
+            _currentPageIndex = index;
+          });
+        },
+        selectedIndex: _currentPageIndex,
+        elevation: 10,
+        height: 100,
+	labelBehavior: NavigationDestinationLabelBehavior.alwaysShow,
```

Comment:
コピペした結果？タブ文字になっているので半角スペースに変更しよう！
```suggestion
        labelBehavior: NavigationDestinationLabelBehavior.alwaysShow,
```